base/sysimg.jl

```diff
@@ -294,12 +294,12 @@ include("client.jl")
 include("util.jl")
 
 # dense linear algebra
+include("broadcast.jl")
+importall .Broadcast
```

Comment:
probably before the linear algebra comment since broadcast is a bit more general than that